gap-analysis/index.md

```diff
@@ -286,17 +286,53 @@ Such an attribute could be used by an authoring tool, for example, to encode the
 </mrow>
 ```
 
-If  had another meaning, such as an open interval, or an ordered pair, or greatest common divisor, a value other than “point” would be used in the value of “intent”.
+If (0,5) had another meaning, such as an open interval, or an ordered pair, or greatest common divisor, a value other than “point” would be used in the intent attribute.
 
 
-Various proposals have been discussed for the syntax to be supported by the intent attribute, most of which functionally equivalent, but with varying trade-offs in markup ergonomics. Global or local identifiers for referencing subexpressions, as well as the verbosity/repetitiveness of annotations are two such aspects. including “Default intent values” for existing MathML presentation forms, including default names for math operators, default rules for applying operators to arguments. These defaults help reduce the amount of markup needed to specify the intent of an expression. While the full encoding of the content markup for an expression is often complex, and often not included with the presentation, the inclusion of the intent markup for an expression is potentially much simpler, and easier to consume as part of the presentation markup.
+The line segment from the point A' to the point B' could be encoded as:
 
+```xml
+<mover intent="segment($x,$y)">
+  <mrow>
+    <msup arg="x" intent="prime($1)">
```

Comment:
There are three examples that we picked out of a MathCounts problem presented earlier in the document (currently hidden in a 'details' section, but I should probably pull the examples out so they are more prominent. All the potential approaches should have versions of those three examples.

As for this one, I agree that "prime" has confusion due to its math meaning. On the hand, there is an argument to be had as to whether these should use a functional syntax, should be `intent="a-prime"` and `intent="b-prime"` or whether they are needed at all because a perfectly sensible proposal would be that intent only needs to be given when the natural reading of the characters needs to be altered. In this case `&#x2032;` has the (Unicode) name "prime". Since 'A' doesn't have an intent, why does the prime need it? I'm not advocating for that, just pointing out that  `intent="a-prime"` isn't helpful in this case.